core/include/moveit/task_constructor/stages/modify_planning_scene.h

```diff
@@ -41,6 +41,7 @@
 #include <moveit/task_constructor/stage.h>
 #include <moveit/task_constructor/properties.h>
 #include <moveit/task_constructor/type_traits.h>
+#include <geometry_msgs/PoseStamped.h>
```

Comment:
I think this `include` has become obsolete. Maybe replace it with `moveit_msgs/CollisionObject.h`?